checkstyle/suppressions.xml

```diff
@@ -179,6 +179,10 @@
     <!-- Instance -->
     <suppress checks="Javadoc(Method|Type|Variable)" files="com[\\/]hazelcast[\\/]instance[\\/]"/>
 
+    <!-- SPI -->
+    <suppress checks="FileLength"
```

Comment:
The file length suppression can be only added in `suppressions.xml` because checkstyle reports this error on the 1st line of the file which must be the license header.